app/assets/stylesheets/storytime/posts.scss

```diff
@@ -74,3 +74,7 @@
   #draft_last_saved_at{
     margin-left: 25px;
   }
+
+  #delete-blog-post {
```

Comment:
Could we make this a class rather than id, since there will be multiple on the page at one time?